roles/node/tasks/authority_metrics.yml

```diff
@@ -0,0 +1,50 @@
+---
```

Comment:
please move it to another role called metrics, follow the approach similar to supra telegraph and promtail